api/test_resources/boxes.json

```diff
@@ -0,0 +1,1341 @@
+      },
+      "witness": {
```

Comment:
A `witness` is generated by a route's `prover` which typically just parses the response json using generated go types.